sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -69,242 +79,244 @@ class PolicyCertificatesModificationResult(object):
     """
 
     def __init__(self, certificate_thumbprint, certificate_resolution):
-        #type:(str, CertificateModification)->None
+        # type: (str, CertificateModification, AttestationToken) -> None
         self.certificate_thumbprint = certificate_thumbprint
         self.certificate_resolution = certificate_resolution
 
     @classmethod
     def _from_generated(cls, generated):
+        # type: (GeneratedPolicyCertificatesModificationResult) -> AttestationPolicyCertificateResult
         if not generated:
-            return cls
+            return None
         return cls(generated.certificate_thumbprint, generated.certificate_resolution)
 
-class PolicyResult(object):
-    """ PolicyResult represents the result of a :meth:`azure.security.attestation.AttestationAdministrationClient.set_policy` 
-    or :meth:`azure.security.attestation.AttestationAdministrationClient.reset_policy`  API call.
 
-    The `PolicyResult` class is returned as the body of an attestation token from
+class AttestationPolicyResult(object):
+    """AttestationPolicyResult represents the result of a
```

Comment:
nit: for consistency and brevity
```suggestion
    """The result of a
```